Zilon.Core/Zilon.TextClient/Program.cs

```diff
@@ -61,25 +63,66 @@ static async System.Threading.Tasks.Task Main()
 
             do
             {
+                PrintState(uiState.ActiveActor.Actor);
+                Console.WriteLine("Input command:");
                 var inputText = Console.ReadLine();
                 if (inputText.StartsWith("m"))
                 {
+                    var components = inputText.Split(' ');
+                    var x = int.Parse(components[1]);
+                    var y = int.Parse(components[2]);
+                    var offsetCoords = new OffsetCoords(x, y);
+
+                    ISectorMap map = playerActorSectorNode.Sector.Map;
+
+                    var targetNode = map.Nodes.OfType<HexNode>().SingleOrDefault(node => node.OffsetCoords == offsetCoords);
+
                     var taskSource = scope.ServiceProvider.GetRequiredService<IActorTaskSource<ISectorTaskSourceContext>>();
 
                     var moveCommand = scope.ServiceProvider.GetRequiredService<MoveCommand>();
 
-                    var nextMoveNode = playerActorSectorNode.Sector.Map.GetNext(uiState.ActiveActor.Actor.Node).First();
-                    uiState.SelectedViewModel = new NodeViewModel { Node = (HexNode)nextMoveNode };
+                    uiState.SelectedViewModel = new NodeViewModel { Node = targetNode };
 
                     moveCommand.Execute();
                 }
 
+                if (inputText.StartsWith("look"))
+                {
+                    var nextMoveNodes = playerActorSectorNode.Sector.Map.GetNext(uiState.ActiveActor.Actor.Node);
+                    Console.WriteLine("Nodes:");
+                    Console.WriteLine();
+                    foreach (var nextNode in nextMoveNodes)
+                    {
+                        Console.Write(nextNode);
+                        if (playerActorSectorNode.Sector.Map.Transitions.TryGetValue(nextNode, out var _))
+                        {
+                            Console.Write(" t");
+                        }
+                        Console.WriteLine();
+                    }
+                }
+
                 if (inputText.StartsWith("exit"))
                 {
                     break;
                 }
             } while (true);
         }
+
+        private static void PrintState(IActor actor)
+        {
+            Console.WriteLine(new string('=', 10));
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6388157)